web/cobrands/northamptonshire/assets.js

```diff
@@ -55,61 +43,74 @@ var layers = [
       "Restricted Visibility"
   ],
   "layer_name": "Tree",
-  "styleid": '5d481376fe2ad8059ce44ef2',
   "snap_threshold": 0
 },
 {
   "categories": [ "Safety Bollard - Damaged/Missing" ],
-  "layer_name": "Safety Bollard",
-  "styleid": "5d481446fe2ad8059ce44f02",
+  "layer_name": "Safety_Bollard"
 },
-];
-
-var prow_assets = [
 {
   "categories": [ "Bridge-Damaged/ Missing" ],
   "item_name": "bridge or right of way",
-  "layer_name": "BRIDGES",
-  "styleid": "5d48161ffe2ad809d85a7add"
+  "layer_name": "Bridges"
 },
 {
   "categories": [ "Gate - Damaged/ Missing" ],
   "item_name": "gate or right of way",
-  "layer_name": "GATE",
-  "styleid": "5d481906fe2ad8059ce450b4",
+  "layer_name": "Gates"
 },
 {
   "categories": [ "Stile-Damaged/Missing" ],
   "item_name": "stile or right of way",
-  "layer_name": "STILE",
-  "styleid": "5d481a05fe2ad8059ce45121",
+  "layer_name": "Stile"
 },
 {
   "categories": [ "Sign/Waymarking - Damaged/Missing" ],
   "item_name": "waymarking or right of way",
-  "layer_name": "WAYMARK POST",
-  "styleid": "5d481a4ffe2ad809d85a7b90&styleIds=5d481742fe2ad809d85a7b05"
+  "layer_name": "Waymarker"
+},
+{
+  "categories": [
+    "Damaged/Exposed Wiring / Vandalised",
+    "Lamp/Bulb Failure",
+    "Signal Failure",
+    "Signal Failure all out",
+    "Signal Stuck",
+    "Signal Head Failure",
+    "Request Timing Review",
+    "Damaged Control box",
+    "Signal Failure/Damaged - Toucan/Pelican"
+  ],
+  "item_name": "signal or crossing",
+  "layer_name": "Traffic_Signal_Junction"
 },
+{
+  "categories": [ "Pedestrian Barriers - Damaged / Missing" ],
+  "item_name": "pedestrian barrier",
+  "layer_name": "Pedestrian_Barrier"
```

Comment:
This has lost the asset_type area.

Reply:
As above, the pedestrian barriers are now spots rather than area assets.